polars-sql/Cargo.toml

```diff
@@ -0,0 +1,72 @@
+[package]
+name = "polars-sql"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = "1"
+serde_json = { version = "1" }
+sqlparser = { version = "0.15.0" }
+
+[dependencies.polars]
+path = "../polars"
+default-features = false
+features = [
+  "dynamic_groupby",
+  "zip_with",
+  "simd",
+  "lazy",
+  "strings",
+  "temporal",
+  "random",
+  "object",
+  "csv-file",
+  "fmt",
+  "performant",
+  "dtype-full",
+  "rows",
+  "private",
+  "round_series",
+  "is_first",
+  "asof_join",
+  "cross_join",
+  "dot_product",
+  "concat_str",
+  "row_hash",
+  "reinterpret",
+  "decompress-fast",
+  "mode",
+  "extract_jsonpath",
+  "lazy_regex",
+  "cum_agg",
+  "rolling_window",
+  "interpolate",
+  "list",
+  "rank",
+  "diff",
+  "pct_change",
+  "moment",
+  "arange",
+  "true_div",
+  "dtype-categorical",
+  "diagonal_concat",
+  "horizontal_concat",
+  "abs",
+  "ewma",
+  "dot_diagram",
+  "dataframe_arithmetic",
+  "json",
+  "string_encoding",
+  "product",
+  "ndarray",
+  "series_from_anyvalue",
+  "avro",
+  "parquet",
+  "ipc",
+  "is_in",
+  "serde",
+]
+
+[workspace]
```

Comment:
Any reason `polars-sql` defines its own workspace? It breaks building from sdist for `py-polars` now that `polars-sql` is included in it. See

* https://github.com/conda-forge/polars-feedstock/pull/50
* https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=573525&view=logs&jobId=6920d3d3-ef74-5584-b309-e8e970b1eb3b&j=6920d3d3-ef74-5584-b309-e8e970b1eb3b&t=bc2ce270-cdc3-50c0-3478-1273af1baa71

Reply:
No particular reason. Good observation. I shall add that crate to the same workspace. :+1: 

So for an `sdist` package all local crates must share a workspace?

Reply:
> So for an sdist package all local crates must share a workspace?

At the moment I think so, it's a quirk from how maturin implements sdist with local path dependencies, I hope we can lift the limitation in the future.